webgpu.yml

```diff
@@ -436,9 +436,15 @@ enums:
       TODO
     entries:
       - name: undefined
+        value: 0x0001
+        doc: |
+          TODO
```

Comment:
We don't need `undefined` here; we only have it in Dawn as an intermediate deprecation before we remove it. In this repo we are just trying to converge to 1.0. So we can skip the deprecation.
```suggestion
```